tests/RobotFramework/tests/cumulocity/custom_operation/custom_operation_command/c8y_Command_3

```diff
@@ -0,0 +1,4 @@
+[exec]
+topic = "${.topic.root_prefix}/custom/topic/one"
```

Comment:
What does `${.topic.root_prefix}` refer to?

It would be more common to be able to expand the mapper prefix (e.g. `c8y` or whether the cloud prefix etc.), and to avoid confusion with the `topic.root_topic` value (which is currently the local thin-edge.io topic prefix, e.g. `te`).

Reply:
Sorry I just read the other code, and could answer the questions.

We should rename the variable as follows:

`${.topic.root_prefix}` => `${.bridge.topic_prefix}`

The `.bridge.topic_prefix` then aligns with the tedge.toml value `c8y.bridge.topic_prefix`, where the leading `c8y.` prefix is not required as the mapper already knows what context it is (as the mapper is started with `tedge-mapper c8y`)

Reply:
resolved by c1311aaf198b4b93fd96ab439ab77f816945e7e3

Reply:
`${.bridge.topic_prefix}` sounds really ad-hoc and not extensible in the future.

I would be explicit with `${.config.c8y.bridge.topic_prefix}` and more general with the ability to inject any config setting. Surely more useful for the script rather than the topic on which commands are expected, I agree. However,  making the context explicit avoid consistency and extensibility issues.

Injecting any config value might be a bit too involved for this PR. Hence, it's okay to start with a simple string replacement (as currently implemented) but with an explicit config context (i.e. `${.config.c8y.bridge.topic_prefix}`).

Reply:
Though the `${.bridge.topic_prefix}` value is actually the prefix is used by the cloud  "profile" used by the mapper. And since the custom operation templates are under the context of the mapper's cloud already, the value does not need to "c8y" context helper.

Reply:
> the value does not need to "c8y" context helper.

Yes. My concern is that this looks like as a path templating system while it is not, just variable substitution using `${.bridge.topic_prefix}` as a variable name.

Reply:
Though doesn't the the property name `topic` suggest otherwise?

Reply:
Sorry I misunderstood your point. I've opened a new discussion with the code in question. https://github.com/thin-edge/thin-edge.io/pull/3189/files#r1813004355

Reply:
@reubenmiller And I misunderstood yours ;-)

After an offline discussion, I agree that it makes sense to use the *bridge* config and not the *tedge* config as root of the substitution in `${.bridge.topic_prefix}`.

For the implementation, what is done is working and can be taken as fist step - even if not extensible.